src/manage.c

```diff
@@ -4226,7 +4226,8 @@ target_osp_ssh_credential (target_t target)
   if (credential)
     {
       iterator_t iter;
```

Comment:
I think naming this something more descriptive like "credentials", could have
helped avoid the error.